### PR TITLE
Change wording in VPS providers document

### PR DESCRIPTION
**Pull request by Vexed01**
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
Looking at this it doesn't make complete sense. The company and tier are listed, then it says that those companies have free tier VPSes which has already been mentioned.

https://red-discordbot--4050.org.readthedocs.build/en/4050/host-list.html#free-hosting

_Stupid author and reviewer of #3916..._

### CODE REVIEW

**Comment by Drapersniper** [comment]
I somewhat disagree with the wording here.

The changes imply that AWS is suitable to small bots, rather than the original stating that free tiers are avaliable and those free tiers are suitable.

If anything I'd change:

`AWS EC2 Free Tier <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
AWS EC2 is not *always* free—it's a 12 month free trial.

to

`AWS EC2 <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
AWS EC2 is not *always* free—it's a 12 month free trial.

File: docs/host-list.rst [hunk 1]
```diff
@@ -79,10 +79,10 @@ server. Any modern hardware should work 100% fine.
...
 ------------
 
-`Google Cloud Compute Free Tier <https://cloud.google.com/free/docs/gcp-free-tier>`_,
-`Oracle Cloud Compute Always Free <https://oracle.com/cloud/free/#always-free>`_ and
-`AWS EC2 Free Tier <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
-AWS EC2 is not *always* free—it's a 12 month free trial.
+`Google Cloud Compute Engine <https://cloud.google.com/free/docs/gcp-free-tier>`_,
+`Oracle Cloud Compute <https://oracle.com/cloud/free/#always-free>`_ and
+`AWS EC2 <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
+Note AWS EC2 is not *always* free—it's a 12 month free trial.
```
Comment (Jackenmen) [inline docs/host-list.rst]: I think this note would be clearer like this:
```suggestion
Note: AWS EC2's free tier does not last forever - it's a 12 month trial.
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]

File: docs/host-list.rst [hunk 2]
```diff
@@ -79,10 +79,10 @@ server. Any modern hardware should work 100% fine.
...
-`Google Cloud Compute Free Tier <https://cloud.google.com/free/docs/gcp-free-tier>`_,
-`Oracle Cloud Compute Always Free <https://oracle.com/cloud/free/#always-free>`_ and
-`AWS EC2 Free Tier <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
-AWS EC2 is not *always* free—it's a 12 month free trial.
+`Google Cloud Compute Engine <https://cloud.google.com/free/docs/gcp-free-tier>`_,
+`Oracle Cloud Compute <https://oracle.com/cloud/free/#always-free>`_ and
+`AWS EC2 <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
+Note: AWS EC2's free tier does not last forever - it's a 12 month trial.
 Additionally, new Google Cloud customers get a $300 credit which is valid
 for 12 months.
```
Comment (Jackenmen) [inline docs/host-list.rst]: It doesn't look that good without any line breaks:
image

Perhaps we could add some?
image

```suggestion
| `Google Cloud Compute Engine <https://cloud.google.com/free/docs/gcp-free-tier>`_,
  `Oracle Cloud Compute <https://oracle.com/cloud/free/#always-free>`_ and
  `AWS EC2 <https://aws.amazon.com/free/>`_ have free tier VPSes suitable for small bots.
| **Note:** AWS EC2's free tier does not last forever - it's a 12 month trial.
| Additionally, new Google Cloud customers get a $300 credit which is valid
for 12 months.
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]

**Review by Jackenmen — APPROVED** [review]
Looks good to me, thanks for the PR.

**Pull request merged**